StandardLayout/css/stelter-standard-a0000000-r0.css

```
@@ -199,25 +197,23 @@
   	#skip-link {position:absolute; top:10px; left:10px; padding:5px; z-index:1;}
    
 	/* General Content */
-	.sub-middle-section p, .sub-middle-section table, .sub-middle-section tr, .sub-middle-section td, .sub-middle-section form, .sub-middle-section label, .sub-middle-section blockquote {font:inherit; color:inherit;}
+	.sub-middle-section p, .sub-middle-section table, .sub-middle-section tr, .sub-middle-section td, .sub-middle-section form, .sub-middle-section label, .sub-middle-section blockquote {}
 	.sub-middle-section p, .sub-middle-section table, .sub-middle-section form {margin:0 0 1.5rem;}
 	.sub-middle-section blockquote {margin:0 20px 1.5rem; padding:3px 15px; border-left:4px solid #ddd;}
     .sub-middle-section p.intro-copy {}
 
 	/* Match the color, margins/padding, font size, etc... to the client's site for all headers */
-	h1, .h1, h2, .h2, h3, .h3, h4, .h4 {} /* Headings Reset */
+	h1, .h1, h2, .h2, h3, .h3, h4, .h4 {margin-top:0;} /* Headings Reset */
 	.stl-page-header {font-family:inherit; font-size:3.2rem; color:inherit;} /* Page header (not always used) */
 	.stl-page-title {font-family:inherit; font-size:3.2rem; color:inherit;} /* Page title styles */    
-        .subpage .stl-page-title {margin-bottom:5px;}
-	.stl-page-subtitle {font-family:inherit; font-size:2.4rem; color:inherit;} /* Sub Title styles */
+	.stl-page-subtitle {font-family:inherit; font-size:2.4rem; color:inherit; font-weight:700;} /* Sub Title styles */
         .subpage #stl-cms .stl-page-subtitle {margin-bottom:30px;}
-	.sub-middle-section h3, .main-copy h3, .main-copy h2 {font-family:inherit; font-size:2.0rem; color:inherit; margin-top:0;}
+	.sub-middle-section h3, .main-copy h3 {}
 
 	/* If interior pages use the two column layout */
 	.interior-one-col .stl-page-title {text-align:left; max-width:990px; margin:0 auto 10px; padding:0 15px;}
 	.interior-one-col .stl-page-subtitle {text-align:left; max-width:990px; margin:0 auto 15px; padding:0 15px;}
-	.interior-two-col .stl-page-title {width:100%; flex:0 0 auto;}
-	.interior-two-col .stl-page-subtitle {width:100%; flex:0 0 auto; margin:0 0 15px;}
+	.interior-two-col header {width:100%; flex:0 0 auto;}
```

add:
.interior-one-col header {...}

---

Looking more into the inspector and going back and forth between one and two column layouts, looks like this is what that section needs to be updated to:

```suggestion
	/* If interior pages use the one column layout */
	.interior-one-col header {max-width:990px; margin:0 auto 10px; padding:0 15px;}

	/* If interior pages use the two column layout */
	.interior-two-col header {width:100%; flex:0 0 auto;}
```